llvm/docs/GitHub.rst

```diff
@@ -214,6 +214,13 @@ commonly used first:
   request will understand that you're rebasing just your patches, and display
   this result correctly with a note that a force push did occur.
 
+.. note::
+   When using `Squash and merge` in LLVM’s GitHub web interface, GitHub will
```

Comment:
"when using squash and merge" sounds like this is a choice but there is no choice if you are in llvm-project. I would say:
"The LLVM Project monorepo uses the "Squash and Merge" setting for PRs. This means that...."

Then there's no choice there's no "oh but am I using squash and merge". If you are here in llvm, you are already using it.

Reply:
Also, as I did in the other page, I would state the negative part.

"GitHub will use W and X. It will **not** use anything from Y and Z."